tests/test_service.py

```diff
@@ -0,0 +1,258 @@
+                'TestOption2': 'Incorrect_1-2'
+            }
+        config['TestSection2'] = \
+            {
+                'TestOption1': 'Incorrect_2-1',
+                'TestOption2': 'Incorrect_2-2'
+            }
+
+        result = EpoService._get_option(
+            config=config,
+            section='TestSection1',
+            option='TestOption1',
+            default_value='DefaultValue'
+        )
+
+        self.assertEqual(correct_value, result)
+
+
+    def test_getoption_default(self):
+        config = ConfigParser()
+        default_value = 'DefaultValue'
+
+        config['TestSection1'] = \
+            {
+                'TestOption1': 'Correct',
+                'TestOption2': 'Incorrect_1-2'
+            }
+        config['TestSection2'] = \
+            {
+                'TestOption1': 'Incorrect_2-1',
+                'TestOption2': 'Incorrect_2-2'
+            }
+
+        result = EpoService._get_option(
+            config=config,
+            section='TestSection1',
+            option='TestOption3',
+            default_value=default_value
+        )
+
+        self.assertEqual(default_value, result)
+
+
+    def test_getbooleanoption(self):
+        config = ConfigParser()
+
+        config['TestSection1'] = \
+            {
+                'TestOption1': 'True',
+                'TestOption2': 'False'
+            }
+        config['TestSection2'] = \
+            {
+                'TestOption1': 'False',
+                'TestOption2': 'False'
+            }
+
+        result = EpoService._get_boolean_option(
+            config=config,
+            section='TestSection1',
+            option='TestOption1',
+        )
+
+        self.assertTrue(result)
+
+
+    def test_getbooleanoption_default(self):
+        config = ConfigParser()
+
+        config['TestSection1'] = \
+            {
+                'TestOption1': 'True',
+                'TestOption2': 'True'
+            }
+        config['TestSection2'] = \
+            {
+                'TestOption1': 'True',
+                'TestOption2': 'True'
+            }
+
+        result = EpoService._get_boolean_option(
+            config=config,
+            section='TestSection1',
+            option='TestOption3',
+        )
+
+        self.assertFalse(result)
+
+
+class TestEpoRequestCallback(BaseClientTest):
+
+    def test_eporequestcallback(self):
+
+        mock_dxl_client = MockDxlClient()
+        with MockServerRunner() as server_list:
+            server_info = server_list[0]
+            test_topic = "/test/topic"
+
+            epo = dxleposervice._epo._Epo(
+                server_info[SERVER_INFO_SERVER_NAME_KEY],
+                LOCALHOST_IP,
+                server_info[SERVER_INFO_SERVER_PORT_KEY],
+                TEST_USER,
+                TEST_PASSWORD,
+                False
+            )
+
+            epo_topics = {test_topic: epo}
+
+            class TestFirstInstanceCallback(dxleposervice.app._EpoRequestCallback):
```

Comment:
Do you need to create a subclass of `_EpoRequestCallback` for this test or could you just create an instance of `_EpoRequestCallback` directly? It doesn't appear to me that the subclass was specializing anything from the base class.